apl-tools/src/main/java/com/apollocurrency/aplwallet/apl/tools/impl/heightmon/HeightMonitor.java

```diff
@@ -85,97 +52,12 @@ public void start() {
 
     public void stop() {
         try {
+            container.shutdown();
             executor.shutdown();
```

Comment:
Should we shutdown executor at first and container at second ??

Reply:
fixed